npm_and_yarn/lib/dependabot/npm_and_yarn.rb

```diff
@@ -165,6 +165,9 @@ module NpmAndYarn
       REQUIREMENT_NOT_PROVIDED: /(?<dep>.*)(.*?)doesn't provide (?<pkg>.*)(.*?), requested by (?<parent>.*)/
     }.freeze, T::Hash[String, Regexp])
 
+    # registry returns malformed response
+    REGISTRY_NOT_REACHABLE = /Received malformed response from registry for "(?<ver>.*)". The registry may be down./
```

Comment:
should we retry if these are transient?

Reply:
@abdulapopoola , doesn't seem transient in retry context,

I tried to access the package in update (https://registry.npmjs.org/teste-react-jv/-/teste-react-jv-0.1.0.tgz) and it was not found , may be the devs removed it temporarily. in any case, the registry returns malformed response which is generic for similar errors.